msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,15 +176,125 @@ The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs usin
+
+The following is the flow diagram of a cached app when it's added to the meeting stage:
+
+:::image type="content" source="../assets/images/saas-offer/cached-launch-app.png" alt-text="This screenshot shows the flow of the cached launch of the app in meeting stage.":::
+
+After you enable app caching, the webview that is used to host the embedded app is reused as users navigate to different instances of the app within a window.
+
+The webview of the app remains in the Document Object Model (DOM). The webview is hidden when the users go out of the app and shown when the users return to the app. When the app is cached, any audio that is playing is muted.
+
+> [!NOTE]
+> If the app caching is not enabled, the webview is recreated every time the users go out and return to the app.
+
+Following are the parameters to control the conditions for the apps to be added or removed from the cache:
+
+* Only one app is supported in the cache for app caching. When there's more than one app with app caching in a meeting, then the least recently used app is removed from the cache.
+* When the app is cached, the memory (working set) usage must not exceed 225 MB.
+* If the user doesn't return to the app within 20 minutes, the app is removed from the cache.
+* The maximum time for Teams to receive the `readyToUnload` signal from the app is 30 seconds.
+* The app has one-minute grace period to get the memory usage down after the app is cached.
+* App caching is disabled if the system memory is less than 4 GB or the available free memory is less than 1 GB (512 MB on Mac).
+* Side panel is the only supported FrameContext for app caching in meetings.
+* When the app is cached, CPU usage must not exceed 5%.
+* When the app is cached, the number of SDK requests shouldn't exceed five for every 12 seconds.
+* The cache state is monitored every 12 seconds and the apps that don’t meet the requirements are removed from the cache.
+
+### Code example
+
+The following code snippet is an example of `registerBeforeUnloadHandler` and `registerOnLoadHandler` APIs:
+
+```javascript
+microsoftTeams.registerBeforeUnloadHandler((readyToUnload) => { 
+console.log("got beforeunload from TEAMS"); 
+// dispose resources and then invoke readyToUnload 
+readyToUnload(); 
+return true; 
+}); 
+
+microsoftTeams.registerOnLoadHandler((data) => { 
+console.log("got load from TEAMS", data.contentUrl, data.entityId); 
+// use contentUrl to route to correct page 
+// invoke notifySuccess when ready  
+microsoftTeams.appInitialization.notifySuccess(); 
+}); 
+```
+
+### Limitations
+
+The following are the limitations for app caching:
+
+* Single-page apps that use client-side routing for page navigation can benefit from app caching. It's recommended that the same domain is used across all contexts of your app launch. For example, using *bar.foo.com* for chats and *baz.foo.com* for personal app isn't recommended as you need to go to the new domain in the load handler.
```

Comment:
Update the last line out so it is less confusing. Indicate that App Caching is not supported in chat, personalApp context yet and support for this will come later. Or leave out the last line entirely

Reply:
Since the example is talks about chant and personalapp which is not supported, hence removing the example.